pkg/client/unversioned/extensions.go

```diff
@@ -130,24 +130,24 @@ func NewExtensionsOrDie(c *Config) *ExtensionsClient {
 
 func setExtensionsDefaults(config *Config) error {
 	// if experimental group is not registered, return an error
-	g, err := latest.Group("extensions")
-	if err != nil {
-		return err
+	if !extlatest.IsEnabled() {
```

Comment:
> Apart from what liggitt said, another reason to use the central latest is to minimize the import graph.
> 
> Sorry I didn't catch up with this PR earlier. Could you hold on for a day? I want to discuss it with @lavalamp.

Yes, we can hold for a day.

I would rather know my true dependencies via an import graph than hide them behind a lookup.  This code won't work properly without its dependency and I don't want to discover that some linkage is broken at runtime if I have the options of knowing at compile time.